config.default.yaml

```diff
@@ -62,18 +62,28 @@ electricity:
 atlite:
   nprocesses: 4
   cutouts:
+    # use 'base' to determine geographical bounds and time span from config
+    # base: 
+      # module: era5  
```

Comment:
This seems unnecessary and only be used by the `mock_snakemake(...)` in `build_cutout(...)`. Is this a remnant from dev/testing?

Reply:
Two thoughts in opposite directions.

1. Yes, it is convinient, but it is not used. That's the differenc between the base network and base cutout. Is it based on `era5` or on `sarah`? None of the renewables are defined for it.

2. With the new `atlite` version we can combine `cutouts` from different datasets unless the variable overlap, right? So nothing is really stopping us from having a true `base` cutout with `era5` and `sarah` data in it? We could merge the two separate cutouts and then switch all renewables profiles to the base cutout. The need for separation is gone.

Reply:
Good points
1. Yes, for now I would leave it just there, so we can compare the results properly the 'master' version
2. Exactly (if variables overlap the first mentioned module is prioritized). The only advantage in separating them is the keeping the higher resolution in the sarah cutout (0.2 x 0.2). But I think it's good to go with an overall 0.25 x 0.25 resolution

Reply:
How about we take this idea and deal with it in a separate issue? Let's focus on getting the new atlite version in.
(Sorry, I tend to get side tracked as well).

Reply:
I totally agree. (no worry, it's good to discuss things)